server/src/instant/admin/routes.clj

```diff
@@ -340,6 +344,61 @@
                      :headers {"authorization" (str "Bearer " admin-token)
                                "app-id" (str counters-app-id)}}))
 
+
+;; ---
+;; Storage
+
+(defn signed-download-url-get [req]
+  (let [{app-id :app_id} (req->admin-token! req)
+        filename (ex/get-param! req [:params :filename] string-util/coerce-non-blank-str)
```

Comment:
Nit: Looks like apart form the perm check, these are going to be very similar to the client endpoints. i.m.o it may make sense to have them as general-purpose functions, that access some kind of 'perms' object. This way, we can use them in both places